explorer/src/components/common/Tooltip.tsx

```diff
@@ -8,17 +8,20 @@ import { FC, useCallback, useState } from 'react'
 interface TooltipProps {
   text: string | React.ReactNode
   children: React.ReactNode
+  direction?: 'top' | 'bottom' | 'left' | 'right' // New prop for tooltip direction
 }
 
-export const Tooltip: FC<TooltipProps> = ({ text, children }) => {
+export const Tooltip: FC<TooltipProps> = ({ text, children, direction = 'top' }) => {
   const [isVisible, setIsVisible] = useState(false)
 
   const toggleVisibility = useCallback(() => setIsVisible(!isVisible), [isVisible])
 
   return (
     <div className='group relative flex flex-col items-center'>
       {isVisible && (
-        <div className='absolute top-full z-10 mt-2 w-auto rounded-md bg-purpleAccent p-2 text-sm text-white shadow-lg'>
+        <div
```

Comment:
If you keep the mouse over an element the tooltip keeps blinking intermittently

Reply:
Thanks, indeed, this issue is now fix